### ISSUE TITLE
Update Maven repos list for productized images

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis#2687]
## This is a...

<pre><code>
[X] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>
Currently the productised templates contains these list of repos:
```
 maven:
        repositories:
          01_maven_central: https://repo1.maven.org/maven2
          02_redhat_ea_repository: https://maven.repository.redhat.com/earlyaccess/all
          03_jboss_ea: https://repository.jboss.org/nexus/content/groups/ea
```

But in fuse-ignite-we replace 02 and 03 with:

```
maven_repo_redhat="https://maven.repository.redhat.com/ga/"
maven_repo_jboss="https://repository.jboss.org/"
```

So, my question is, whether we shouldn't just put these two repos into the original above ?

@dsimansk @paoloantinori any thoughts on this ?

**Comment by dsimansk** [comment]
Sounds about right, currently `maven.repository.redhat.com/ga` is the most important. Because GA artifact will go into it. I've had discussion with Marek about `/earlyaccess/all` that should be a conjuction of both `/ga` and `/earlyaccess`, but I have no mean to verify it currently.

Tl;Dr: `maven.repository.redhat.com/ga` is what we need for GA artifacts currently.

**Comment by paoloantinori** [comment]
Hi @rhuss is this still relevant?

**Comment by rhuss** [comment]
Yes, it is. The hack in https://github.com/syndesisio/fuse-ignite-install/blob/e3cad05e14efc578b766ee00987c674b94b9c586/release.sh#L207-L209

is just to replace the repository and was only introduced in the shell script because the template couldn't be changed anymore because of code-freeze. The same for all the other hacks in `release.sh` which do patching of the productised templates.

IMO all of the replacement steps in release.sh should be removed and the fix included in the templates directly.

**Comment by rhuss** [comment]
And we now already can distinguish for productised builds between OSO (hosted) and OCP (on-premise) templates.

**Comment by paoloantinori** [comment]
Okay than, to not incur in the same issue we had in past that forced us to work in the `release.sh` script I'm marking this as a blocker, so that we will address it for this release.

thanks

**Comment by rhuss** [comment]
Fixed with https://github.com/syndesisio/syndesis/pull/3007
